packages/maskbook/src/plugins/Trader/UI/trader/TradeForm.tsx

```diff
@@ -199,6 +200,10 @@ export function TradeForm(props: TradeFormProps) {
     const validationMessage = useMemo(() => {
         if (inputTokenTradeAmount.isZero() && outputTokenTradeAmount.isZero())
             return t('plugin_trader_error_amount_absence')
+        if (inputTokenTradeAmount.isLessThan(MINIMUM_AMOUNT))
```

Comment:
WRONG! Please recheck how is the trade amount made. You cannot compare it with a fraction number.

Reply:
BigNumber can handle data accuracy